docker/images/dev-docker/README.md

```diff
@@ -2,22 +2,22 @@
 
 ## Simple Tags
 
--	[`latest` (*docker/images/skycoindev-cli/Dockerfile*)](https://github.com/skycoin/skycoin/tree/develop/docker/images/skycoindev-cli/Dockerfile)
+-	[`dind` (*docker/images/devdocker/Dockerfile*)](https://github.com/skycoin/skycoin/tree/develop/docker/images/devdocker/Dockerfile)
 
-# Skycoin development image
+# Skycoin development image including [docker in docker](https://hub.docker.com/_/docker/)
 
 This image has the necessary tools to build, test, edit, lint and version the Skycoin
 source code.  It comes with the Vim editor installed, along with some plugins
-to ease go development and version control with git.
+to ease go development and version control with git, besides it comes with docker installed.
 
 # How to use this image
 
 ## Initialize your development environment.
 
 ```sh
 $ mkdir src
-$ docker run --rm \
-    -v src:/go/src skycoin/skycoindev-cli \
+$ docker run --privileged --rm \
```

Comment:
@stdevNorge I am not sure of whether `--privileged` flag is needed for commands not related to `dind` . Could you please try without it ? If Skycoin commands work , then revert changes like this one.

Reply:
That is correct, <code>--privileged</code> flag is not needed for commands not related to <code>dind</code> . I fix the README right now.